composer.json

```
@@ -6,20 +6,20 @@
     "sort-packages": true
   },
   "require": {
-    "ongr/elasticsearch-dsl": "^6.0.1 || ^7.0",
-    "pimcore/pimcore": "^6.9 || ^10.0",
-    "symfony/config": "^4.3 || ^5.2.0",
-    "symfony/console": "^4.3 || ^5.2.0",
-    "symfony/dependency-injection": "^4.3 || ^5.2.0",
-    "symfony/event-dispatcher": "^4.3 || ^5.2.0",
-    "symfony/http-foundation": "^4.3 || ^5.2.0",
-    "symfony/http-kernel": "^4.3 || ^5.2.0",
-    "symfony/messenger": "^4.3 || ^5.2.0",
-    "symfony/routing": "^4.3 || ^5.2.0"
+    "ongr/elasticsearch-dsl": "^7.0",
+    "pimcore/pimcore": "^10.0",
```

```suggestion
    "pimcore/pimcore": "^10.5",
```
for other dependencies like https://github.com/pimcore/advanced-object-search/pull/169 & https://github.com/pimcore/advanced-object-search/pull/181